diff --git a/node_modules/@antv/f2/es/components/legend/legendView.js b/node_modules/@antv/f2/es/components/legend/legendView.js
index 9afb35a..ddb1144 100644
--- a/node_modules/@antv/f2/es/components/legend/legendView.js
+++ b/node_modules/@antv/f2/es/components/legend/legendView.js
@@ -9,9 +9,10 @@ var Marker = function Marker(_ref) {
   if (type === 'square') {
     return jsx("rect", {
       style: {
-        width: '12px',
-        height: '12px',
-        marginRight: '10px'
+        width: '40px',
+        height: '14px',
+        'border-radius': '2px',
+        marginRight: '24px'
       },
       attrs: {
         fill: color
